sources/light_client.move

```diff
@@ -67,7 +67,15 @@ fun init(_ctx: &mut TxContext) {
 
 /// LightClient constructor. Use `init_light_client` to create and transfer object,
 /// emitting an event.
-public fun new_light_client(params: Params, start_height: u64, trusted_headers: vector<vector<u8>>, start_chain_work: u256, finality: u64, ctx: &mut TxContext): LightClient {
+/// *params: Btc net work params. Check the params module
+/// *start_height:  the height of first trust block
+/// *trusted_headers: List of trusted headers in hex format.
+/// *parent_chain_work: chain_work at parent block of start_height block.
+/// *finality: block height must deeper `finality` should consider as finalized block
```

Comment:
The comment is unclear

Reply:
I just explain this is  finality threshold ;)